src/day8/re_frame_10x/tools/coll.cljs

```diff
@@ -43,10 +43,15 @@
   (reduce
     (fn [ret k]
       (cond
-        (list? ret) (nth ret k)
-        (set? ret)  (if (number? k)
-                      (nth (vec ret) k)
-                      (get ret k))
-        :else       (get ret k)))
+        (or (list? ret) (instance? cljs.core/LazySeq ret))
+        (nth ret k)
+        (set? ret)
+        (if (number? k)
+          (nth (vec ret) k)
+          (get ret k))
+        (map? ret)
```

Comment:
the `map?` case and the `:else` case are the same, so could just leave it to the `:else` case alone